lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,92 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: one_or_more_strings?,
+      )
+    end
+
+    ALLOWED_ERRORS = [
+      "no project was found in any parent directories of",
+      "no YAML file was found in",
+    ].freeze
+
+    register_config_schema SCHEMA.config
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - .github/workflows/one_workflow.yml
+          - .github/workflows/another_workflow.yml
+        ignore:
+          - "job .* needs job"
+          - "unknown Webhook event"
+      YAML
+    end
+
+    # This analyser uses git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyzer_options
+      opts = []
+      opts << "-format"
+      opts << "{{json .}}"
+
+      Array(config_linter[:ignore]).each do |rule|
+        opts << "-ignore"
+        opts << rule
+      end
+
+      if config_linter.key?(:target)
+        opts << "--"
+        opts += Array(config_linter[:target])
+      end
```

Comment:
The "--" option is always acceptable, so this block could be simplified as;

```suggestion
      opts << "--"
      Array(config_linter[:target]).each { |v| opts << v }
```

Reply:
Also, there is no test case of multiple targets. Please add the test.

Reply:
I changed it like you suggested, although I find a trailing `--` without any targets in the command line a bit weird looking. It works though.

Reply:
> Also, there is no test case of multiple targets. Please add the test.
Ok, working on it.

Reply:
Multiple targets test case added.